examples/dht_node.rs

```diff
@@ -84,7 +88,7 @@ fn main() {
     // incoming packet.
     let server_obj_c = server_obj.clone();
     let handler = stream.for_each(move |(packet, addr)| {
-        println!("recv = {:?}", packet.clone());
+        //println!("recv = {:?}", packet.clone());
```

Comment:
don't comment this line

Reply:
send_ping_req() is moved into send_pings() to avoid deadlock